lib/middleman-cdn/cdns/cloudflare.rb

```diff
@@ -23,6 +23,7 @@ def self.example_configuration_elements
       end
 
       def invalidate(options, files, all: false)
+        options[:invalidate_zone_for_many_files] = true unless options.has_key?(:invalidate_zone_for_many_files)
```

Comment:
`has_key?` => `key?`. re: http://batsov.com/articles/2013/08/21/the-elements-of-style-in-ruby-number-9-hash-number-has-key-and-hash-number-has-value-are-deprecated/